strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-pypi-layout-provider/src/main/java/org/carlspring/strongbox/domain/PypiMetadataDao.java

```diff
@@ -0,0 +1,95 @@
+package org.carlspring.strongbox.domain;
+
+import javax.inject.Qualifier;
+import java.lang.annotation.*;
+
+public class PypiMetadataDao {
+
+    @PypiMetadataKey(name = "Metadata-Version")
```

Comment:
Clarification: Is the key matched using ignore case? Is `metadata-version` the same as `MeTAdata-VersioN`?

Reply:
It wasn't case insensitive, but I changed it :)

Reply:
Thanks a lot! :)